CRM/Core/Payment/iATSServiceACHEFT.php

```diff
@@ -73,6 +73,10 @@ static public function &singleton($mode, &$paymentProcessor, &$paymentForm = NUL
 
   protected function getDirectDebitFormFields() {
     $fields = parent::getDirectDebitFormFields();
+    if ($fields[0] == 'account_holder') {
+      array_shift($fields);
+      $fields[] = 'account_holder';
+    }
```

Comment:
It looks like you're changing the account_holder to the end of the form field list, is that right?

Reply:
@adixon That's the idea yep. It seems like the logic order, and it ensures that the ordering of entering information appears properly (according to the image). CiviCore seems to drop that as the first element so that's why I had to do some funky work to switch it.